conda-store-server/conda_store_server/plugins/plugin_manager.py

```diff
@@ -0,0 +1,32 @@
+# Copyright (c) conda-store development team. All rights reserved.
+# Use of this source code is governed by a BSD-style
+# license that can be found in the LICENSE file.
+
+import pluggy
+
+from conda_store_server.exception import CondaStorePluginNotFoundError
+from conda_store_server.plugins import BUILTIN_PLUGINS, types
+
+
+class PluginManager(pluggy.PluginManager):
+    def get_lock_plugins(self) -> dict[str, types.TypeLockPlugin]:
```

Comment:
The "get all" uses `get_*`, but the "get one" does not use the `get_` prefix. I think it would be nicer to have a consistent naming scheme for these methods.